services/apps/alcs/src/common/authorization/portal-auth-guard.service.ts

```diff
@@ -25,21 +25,20 @@ export class PortalAuthGuard implements CanActivate {
 
   constructor(
     @Inject(KEYCLOAK_INSTANCE)
-    private singleTenant: Keycloak,
+    singleTenant: Keycloak,
     @Inject(KEYCLOAK_CONNECT_OPTIONS)
     private keycloakOpts: KeycloakConnectConfig,
     @Inject(KEYCLOAK_LOGGER)
-    private logger: Logger,
-    private multiTenant: KeycloakMultiTenantService,
-    private reflector: Reflector,
+    logger: Logger,
+    reflector: Reflector,
     private clsService: ClsService,
     private userService: UserService,
   ) {
     this.keyCloakGuard = new KeyCloakRoleGuard(
       singleTenant,
       keycloakOpts,
       logger,
-      multiTenant,
+      null!,
```

Comment:
Is this an intensional change?

Reply:
Yes, its a bit nasty but it seems like the multi tenant package of nest keycloak broke at some point and fails to import. The good news is that we don't use multi-tenant so not passing it doesn't seem to impact anything.